index.js

```diff
@@ -68,15 +83,17 @@ const changeImage = function() {
       screenSaverImageOne.style.opacity = 1;
     }
 
-    setTimeout(changeImage, 30000); // 30000 = 30 sekunder
+    screenSaverTime.textContent = String(h).padStart(2, '0') + ":" + String(m).padStart(2, '0');
+
+    setTimeout(changeImage, 60000); // 30000 = 30 sekunder
```

Comment:
why has this been changed?

Reply:
Oh, I wanted to have the image change only once a minute.

Reply:
maybe yet another setting?

Reply:
could say that in the Readme and then let it be up to the user if they want faster and wanna pay for the service